src/FixedDataTableRow.js

```diff
@@ -216,26 +216,6 @@ class FixedDataTableRow extends React.Component {
     );
   }
 
-  // shouldComponentUpdate(nextProps) {
```

Comment:
Why you removed this shouldComponentUpdate ?

Reply:
When I combined FixedDataTableRow and FixedDataTableRowImpl at that time I have to also take care of this shouldComponentUpdate .So I updated it and removed the old shouldComponentUpdate .